utils/events/src/test/java/io/smallrye/config/events/ChangeEventNotifierTest.java

```diff
@@ -26,9 +28,14 @@
 
     @Deployment
     public static WebArchive createDeployment() {
+
```

Comment:
Why not SmallRye Config?

Given Thorntail MP Config uses SmallRye Config internally, what does using Thorntail offer?

Reply:
Did not really think about that. Just used code from previous PR to try and see why the build is failing.

Reply:
Seems like it worked and the build is fine now...